src/DateTime.cc

```diff
@@ -359,6 +359,14 @@ dafBase::DateTime::DateTime(double date, DateSystem system, Timescale scale) {
 dafBase::DateTime::DateTime(int year, int month, int day,
                             int hr, int min, int sec, Timescale scale) {
 
+    int const minYear = 1902;
+    int const maxYear = 2261;
+    if ((year < minYear) || (year > maxYear)) {
+        throw LSST_EXCEPT(
+                          lsst::pex::exceptions::DomainError,
```

Comment:
I think you're right. If I had the first argument on the same line then I'd need to indent the remaining lines as shown, but outdenting and removing a line break improves readability